git-cliff-core/src/config.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg(target_os = "macos")]
+use std::env;
```

Comment:
You could also import this inside of the function. I think that would be cleaner.